SnowplowDemo/SnowplowDemo/DemoUtils.m

```diff
@@ -139,24 +139,24 @@ + (void) trackPageViewWithTracker:(SPTracker *)tracker_ {
 + (void) trackScreenViewWithTracker:(SPTracker *)tracker_ {
     SPScreenView *event = [SPScreenView build:^(id<SPScreenViewBuilder> builder) {
         [builder setName:@"DemoScreenName"];
-        [builder setId:@"DemoScreenId"];
+        [builder setEventId:@"DemoScreenId"];
```

Comment:
These should be setScreenId but I'd prefer the old demo application to be deleted rather than this fixed ;-)

Reply:
you are right! I forgot to delete the old demo app! :-o